src/sentry/static/sentry/app/views/substances/substances.jsx

```diff
@@ -220,8 +236,8 @@ const mapStateToProps = state => {
 // TODO: Rename all functions in `mapDispatchToProps` in other files so that they match the action
 // creators name for consistency.
 const mapDispatchToProps = dispatch => ({
-  substanceSearchEntriesGet: (query, groupBy, cursor) =>
-    dispatch(substanceSearchEntriesGet(query, groupBy, cursor)),
+  substanceSearchEntriesGet: (query, groupBy, cursor, isGroupHeader) =>
```

Comment:
I'd like to just mention - to start the discussion - that I think the action creator names convention should be changed in the future. The actions are named `<noun>_<action>` which seems normal for those constants, but it feels unnatural to me that functions are not named `<action>_<noun>`. In our first redux implementation, this was resolved by mapping back to `<action>_<noun>` in `mapDispatchToProps`, but that adds confusion in my mind.

So tl;dr; I think we should use the more natural function names for all action creators, in this case that would be `getSubstanceSearchEntries`.

Reply:
Ok!